### DISCUSSION TITLE
User Input via Console.ReadLine( )

### DISCUSSION THREAD

**Discussion by ghost** [dotnet/interactive#1683] [Q&A]
`Console.ReadLine( );`

doesn't seem to bring up a user input field like it does when using Python in VSCode with the Jupyter notebook. It just keeps on waiting and I have to interrupt the execution of the block.

![2021-11-01 17_52_03-](https://user-images.githubusercontent.com/54975859/139670792-993c1c0d-bccd-4359-a0a9-0d2146e4809a.png)
![2021-11-01 17_52_19-](https://user-images.githubusercontent.com/54975859/139670839-477f1dc5-2dd7-4622-991b-911246ca75c3.png)

Am I missing something here?

**Comment by usirg** [comment]
Ran into the same problem, would like the answer too!

**Reply by jcruzupr** [reply]
I'm having the same issue.  Has anyone come across the solution?

By the way, **Console.ReadLine()** works fine with [Try .NET in the browser](https://dotnet.microsoft.com/en-us/platform/try-dotnet)

![image](https://user-images.githubusercontent.com/1525987/185622114-94415826-fa2d-48f5-a438-bd84342d5dfd.png)

**Comment by jcruzupr** [comment]
Update:

I found a "solution" here: [https://github.com/dotnet/csharp-notebooks/issues/32](https://github.com/dotnet/csharp-notebooks/issues/32)

A pop-up appears where the user can enter the input. For example:
![image](https://user-images.githubusercontent.com/1525987/185645102-b0915b2c-b873-4fe9-bd2b-1df3d64a21bd.png)
